### PR TITLE
RD-1754 Prevent `creating` snapshot from being deleted, updated cypress test accordingly

### PR DESCRIPTION
**Pull request by kubama**
See latest Michael's comment to RD-1754.
System test job run: https://jenkins.cloudify.co/view/UI/job/Stage-UI-System-Test/323/

### CODE REVIEW

File: widgets/snapshots/src/SnapshotsTable.jsx [hunk 1]
```diff
@@ -140,6 +140,7 @@ export default class SnapshotsTable extends React.Component {
 
                     {data.items.map(item => {
                         const isUseful = isSnapshotUseful(item);
```
Comment (qooban) [inline widgets/snapshots/src/SnapshotsTable.jsx:142]: What do you think about changing:
```suggestion
                        const isUsable = isSnapshotUsable(item);
```
?

I know it was not introduced in this PR, but when I look at it now it is misleading.

File: widgets/snapshots/src/SnapshotsTable.jsx [hunk 2]
```diff
@@ -140,6 +140,7 @@ export default class SnapshotsTable extends React.Component {
...
+                        const isRemovable = isUseful || item.status === 'failed';
```
Comment (qooban) [inline widgets/snapshots/src/SnapshotsTable.jsx:143]: Maybe to have `isRemovable` and `isUseful` consts definition more consistent you can either remove `isSnapshotUsable` function or extract this part to a function?

**Review by qooban — COMMENTED** [review]
Added two minor comments.

**Comment by kubama** [comment]
I addressed the comments.
Snapshots and topology widget tests passed on CI.

**Review by qooban — APPROVED** [review]

**Pull request merged**